File: app/src/main/java/com/example/user/exoplayer/player/data/database/VideoUrl.java
package com.example.user.exoplayer.player.data.database;

import android.arch.persistence.room.Entity;
import android.arch.persistence.room.PrimaryKey;

@Entity(tableName = "video")
public class VideoUrl {

    @PrimaryKey(autoGenerate = true)
    private int id;

    private String videoUrl;

    public VideoUrl(String videoUrl) {
        this.videoUrl = videoUrl;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getVideoUrl() {
        return videoUrl;
    }

    public void setVideoUrl(String videoUrl) {
        this.videoUrl = videoUrl;
    }
}

File: app/src/main/java/com/user/exoplayer/player/data/database/VideoUrl.kt
package com.user.exoplayer.player.data.database

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

@Entity(tableName = "video")
class VideoUrl(var videoUrl: String?) {

    @PrimaryKey(autoGenerate = true)
    var id: Int = 0
}
